public_html/wp-content/mu-plugins/blocks/assets/src/sessions/edit.js

```diff
@@ -47,8 +47,14 @@ class SessionsEdit extends Component {
 const sessionsSelect = ( select ) => {
 	const { getEntities } = select( WC_BLOCKS_STORE );
 
+	const sessionArgs = {
+		_embed     : true,
+		meta_key   : '_wcpt_session_type',
+		meta_value : 'session',
```

Comment:
```suggestion
		meta_value : 'session', // Regular sessions only, no breaks/lunch/etc sessions.
```

`'_wcpt_session_type' => 'session'` 

😱 

"session" is a really undescriptive name for that :(

`SHOULD` - It'd be too much of a hassle to go back through all the legacy code and data and fix it, but it might be worth adding a comment here indicating that "session" means "regular", as opposed to "custom" -- also poorly named -- which are breaks.